WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -1052,10 +1052,14 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
             // Only enable reusable block in WP.com sites until the issue
             // (https://github.com/wordpress-mobile/gutenberg-mobile/issues/3457) in self-hosted sites is fixed
             .reusableBlock: isWPComSite,
-            .canViewEditorOnboarding: gutenbergSettings.canViewEditorOnboarding()
+            .canViewEditorOnboarding: self.canViewEditorOnboarding() && !gutenbergSettings.hasLaunchedGutenbergEditor
         ]
     }
 
+    func canViewEditorOnboarding() -> Bool {
+        gutenbergSettings.canViewEditorOnboarding()
+    }
+
```

Comment:
nitpick: I'm fine leaving this in, but accessing `gutenbergSettings.canViewEditorOnboarding()` directly would seem enough, especially since this doesn't provide any more context (e.g. a more descriptive method name).

Reply:
Ignore that, I remember now that this method is needed because it's part of the `PublishingEditor` protocol.